content/en/docs/instrumentation/ruby/sampling.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Sampling
+weight: 6
+---
+
+[Sampling](/docs/concepts/sampling/) is a process that restricts the amount of
+traces that are generated by a system. The Ruby SDK offers several
+[head samplers](/docs/concepts/sampling#head-sampling).
+
+## Default behavior
+
+By default, all spans are sampled, and thus, 100% of traces are sampled. If you
```

Comment:
That's correct, although it's also the same outcome as always sampling (since shouldSample always returns true for the root if you don't configure anything). I can elaborate in the text if you think that will help, though.